pkgs/swift2objc/lib/src/config.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:io';
+
+final defaultTempDir = Directory.systemTemp;
+const defaultTempDirPrefix = 'swift2objc_temp_';
+const symbolgraphFileSuffix = '.symbols.json';
+
+class Config {
+  final List<String> inputFiles;
```

Comment:
After some discussion with the rest of the interop team, we've decided to use `Uri`s instead of `String`s in the public facing APIs. So `inputFiles`, `outputFile`, and `tempDir` should all be `Uri`s. You can convert them to `String`s and manipulate them using package:path if you like (that's what I'll be doing in swiftgen/ffigen), but the public facing type should be a `Uri`.

Reply:
done